tests/tacacs/utils.py

```diff
@@ -51,8 +51,16 @@ def setup_tacacs_client(duthost, creds_all_duts, tacacs_server_ip):
 
     # enable tacacs+
     duthost.shell("sudo config aaa authentication login tacacs+")
-    duthost.shell("sudo config aaa authorization local")
-    duthost.shell("sudo config aaa accounting disable")
+
+    try:
+        duthost.shell("sudo config aaa authorization local")
+    except RunAnsibleModuleFail:
```

Comment:
>except [](http://example.com/codeflow?start=4&length=6)

Instead of using exception, should we check the version or feature available, and config accordingly.